Cargo.toml

```diff
@@ -1,18 +1,20 @@
 [package]
 name = "zero-bounce"
-version = "0.1.2"
+version = "0.1.5"
```

Comment:
that's a steep version increase :))
I assume you forgot to do it for some previous features